### ISSUE TITLE
Ansible Operator playbook / role getting executed twice after initial operator installation, without any modification in resources

### ISSUE THREAD

**Issue by nataraj24** [operator-framework/operator-sdk#6251]
#### What did you do?

Installed sample memcached operator as explained in operator-sdk documentation.   the tasks are idempotent, just creating one sample deployment, secret and configmap.

#### What did you expect to see?
Playbook should be executed only once if its successful.

#### What did you see instead? Under which circumstances?

After operator installation,  the Playbook/role inside operator was executed twice even though playbook execution was successful in first attempt.  Also there were no changes in any of resources to trigger any kind of reconciliation.

Below are status of play book execution after first time installation. Full log is attached.

kubectl logs deploy/memcached-operator-controller-manager -n memcached-operator-system -c manager -f | grep rescued
localhost : ok=3 changed=3 unreachable=0 failed=0 skipped=0 rescued=0 ignored=0
localhost : ok=3 changed=0 unreachable=0 failed=0 skipped=0 rescued=0 ignored=0

[memcached.log](https://github.com/operator-framework/operator-sdk/files/10442245/memcached.log)

#### Environment

**Operator type:**

/language ansible

**Kubernetes cluster type:**

Native K8s cluster. 

operator-sdk version: "v1.25.0", commit: "3d4eb4b2de4b68519c8828f2289c2014979ccf2a", kubernetes version: "1.25.0", go version: "go1.19.2", GOOS: "linux", GOARCH: "amd64"

`$ go version` (if language is Go)

`$ kubectl version`

Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.3", GitCommit:"1e11e4a2108024935ecfcb2912226cedeafd99df", GitTreeState:"clean", BuildDate:"2020-10-14T12:41:49Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}

#### Possible Solution

Check the playbook execution status and if its successful dont execute the playbook again.

**Comment by everettraven** [comment]
Closing this issue as it is a duplicate of #6250 . Please feel free to reopen this issue if you feel it has been closed incorrectly.
